src/Containers/ReqDetailContainer.styles.scss

```diff
@@ -20,3 +21,11 @@ $close-button-width: 35px;
   top: $gutter / 2;
   cursor: pointer;
 }
+
+.tabs-container {
+  display: flex;
+  width: 100%;
+  padding: $gutter;
```

Comment:
$gutter - maybe we can just fix this later

Reply:
PR to fix this https://github.com/saucelabs/network-viewer/pull/17